Reflections/R0 - April 9.md

```diff
@@ -0,0 +1,18 @@
+## 1. What have you accomplished this week? (Please list each accomplishment, and explain)
+
+This week most of what I accomplished was Research into how game maker language works. I looked into it scalability as well and how it handles complex and Abstract data types. I also found a couple of YouTube channels that offer very helpful tutorials, namely Shaun Spalding and FriendlyCosmonaut. While I don't have a whole lot of code to show for this week, I can say that I have a much more confident understanding of how gml works and how best to use it.
+
+
+## 2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+
+The biggest challenge or difficulty that I faced this week was determining a direction to go in. While it is true that I had deliverable two to fall back on so that I had a way to push forward, figuring out some of the smaller details on how to tackle specific issues took a little bit more figuring out. Because of the research that I have now done and the resources I have gathered, I feel more prepared to tackle these smaller steps that I've laid out for myself.
```

Comment:
Great job on gaining a better understanding of the problems you have been facing!